breezedecoration.cpp

```diff
@@ -525,7 +543,7 @@ void Decoration::createButtons()
 
             QColor titleBarColor = this->titleBarColor();
 
-            if ( c->caption().contains(" — Konsole") ) {
+            if ( c->caption().contains(" — Konsole") && m_KonsoleTitleBarColorValid ) {
```

Comment:
Does it mean any window with a " - Konsole" in the caption ? e.g: Firefox, Kate... ?

Reply:
Good question.
It is not a simple minus sign (-) but a long dash (—) and window captions very very rarely add this character, only KDE usually add the application name to the end of the window caption, separated with a long dash.
Now, there is a tiny tiny chance this will cause a transparent titlebar somewhere it's not supposed to. But this chance is so tiny I wouldn't worry about it.
Just to make sure I'll add another conditionto check.